src/common/components/clusterWizard/networkingSteps/NetworkTypeControlGroup.tsx

```diff
@@ -32,36 +32,33 @@ export const NetworkTypeControlGroup = ({
               name={GROUP_NAME}
               isDisabled={isDisabled || !isSDNSelectable}
               value={NETWORK_TYPE_SDN}
-              label={
-                <>
-                  {t('ai:Software-Defined Networking (SDN)')}{' '}
-                  <PopoverIcon
-                    noVerticalAlign
-                    bodyContent={'The classic bullet-proof networking type'}
-                  />
-                </>
-              }
+              label={<>{t('ai:Software-Defined Networking (SDN)')}&nbsp;</>}
             />
           </Tooltip>
         </SplitItem>
-        <SplitItem />
+        <SplitItem>
+          <PopoverIcon
+            bodyContent={'The classic bullet-proof networking type'}
+            buttonStyle={{ top: '-4px' }}
```

Comment:
Is it possible that we don't need `buttonStyle` if we keep `noVerticalAlign` on the `PopoverIcon`?

Plus, I see later instead of `top: -4px` we are using `marginTop: '4px'`. I didn't check if `noVerticalAlign` would work there, but I'd try to keep everything as consistent as possible.

Reply:
I wish it was, but `noVerticalAlign` doesn't really work in this case. At least if we want the icon to stay aligned with the text just as it was before.

Also `top: -4px` and `marginTop: 4px` are fixing two different problems - in one case, the icon is too high, and in the other, it's too low.

Reply:
Understood, if it's not possible to unify them let's keep your changes. Thanks!